azurerm/internal/services/resource/resource_group_resource.go

```diff
@@ -124,7 +124,8 @@ func resourceResourceGroupDelete(d *schema.ResourceData, meta interface{}) error
 		return err
 	}
 
-	deleteFuture, err := client.Delete(ctx, id.ResourceGroup)
+	// @favoretti: This is currently the only supported added value, do we want to add it as an optional parameter to the schema?
+	deleteFuture, err := client.Delete(ctx, id.ResourceGroup, "Microsoft.Compute/virtualMachines,Microsoft.Compute/virtualMachineScaleSets")
```

Comment:
@katbyte @tombuildsstuff Opinions on how to go about this one?

Reply:
Just taken a quick look and for maintaining existing behaviour I think this should to be:
```suggestion
	deleteFuture, err := client.Delete(ctx, id.ResourceGroup, "")
```

Reply:
@jackofallops thank you, commited suggestion. Wasn't sure whether we want it as an empty string or not.
I think we can extrapolate that into a separate argument once this gets in?